### ISSUE TITLE
Document Download

### ISSUE THREAD

**Issue by sgeofrank** [L4GG/timeline#32]
On demand, create a .pdf or .osd where every event is a page in a document so a user can download the entire archive locally as a document.

Allows user to read the entire archive on their kindle or reader. Allows them to reference pages. Allows them to divide and conquer when researching the material.

**Comment by jtbandes** [comment]
I'm thinking a "print-friendly" view would be a good way of doing this.  Also possibly related to #45?

**Comment by anseljh** [comment]
Yes, with #45 in place, it would be (more) straightforward to build whatever other documents we want, including CSV, PDF, whatever. They are essentially just different views of the same data. [`pdfkit`](https://www.npmjs.com/package/pdfkit) might be a good tool for making PDFs.

**Comment by jlegrone** [comment]
This is blocked until #45 is closed.

**Comment by jlegrone** [comment]
I've got a start on this in https://github.com/L4GG/timeline/pull/64.  I'm not committed to handlebars; would be happy with a lightweight client side library too as long as we retain isomorphic rendering.

The key is that we now have a working events json endpoint at `/api/v1/events`.
